server/runner/.env.example

```
@@ -12,4 +12,6 @@ REPL_ID=node-node
 
 CLIENT_URL=http://localhost:3000
 
-OPENAI_API_KEY=very_secret
\ No newline at end of file
+OPENAI_API_KEY=very_secret
+GOOGLE_API_KEY=very_secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)